plugins/woocommerce-blocks/tests/e2e/tests/filter-blocks/basic.block_theme.spec.ts

```diff
@@ -0,0 +1,110 @@
+/**
+ * External dependencies
+ */
+import { test, expect } from '@woocommerce/e2e-playwright-utils';
+
+const wrapperBlock = {
+	name: 'woocommerce/collection-filters',
+	title: 'Collection Filters',
+};
+const filterBlocks = [
+	{
+		name: 'woocommerce/collection-price-filter',
+		title: 'Collection Price Filter',
+		variation: 'Filter Products by Price',
+		heading: 'Filter by Price',
+	},
+	{
+		name: 'woocommerce/collection-stock-filter',
+		title: 'Collection Stock Filter',
+		variation: 'Filter Products by Stock',
+		heading: 'Filter by Stock Status',
+	},
+	{
+		name: 'woocommerce/collection-rating-filter',
+		title: 'Collection Rating Filter',
+		variation: 'Filter Products by Rating',
+		heading: 'Filter by Rating',
+	},
+	{
+		name: 'woocommerce/collection-attribute-filter',
+		title: 'Collection Attribute Filter',
+		variation: 'Filter Products by Attribute',
+		heading: 'Filter Products by Attribute',
+	},
+	{
+		name: 'woocommerce/collection-active-filters',
+		title: 'Collection Active Filters',
+		variation: 'Active Product Filters',
+		heading: 'Active Filters',
+	},
+];
+
+test.describe( 'Filter blocks registration and rendering', async () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test( 'Wrapper block can be inserted through the inserter', async ( {
+		page,
+		editorUtils,
+	} ) => {
+		await editorUtils.openGlobalBlockInserter();
+		await page.getByPlaceholder( 'Search' ).fill( wrapperBlock.title );
+
+		await expect(
+			page.getByRole( 'listbox', { name: 'Blocks' } )
+		).toContainText( wrapperBlock.title );
+
+		await page.getByRole( 'option', { name: wrapperBlock.title } ).click();
+
+		expect(
+			await editorUtils.getBlockByName( wrapperBlock.title )
+		).toBeTruthy();
+	} );
+
+	test( 'Wrapper block contains all filter blocks by default', async ( {
+		editorUtils,
+	} ) => {
+		await editorUtils.insertBlock( {
+			name: wrapperBlock.name,
+		} );
+		for ( const block of filterBlocks ) {
+			expect(
+				await editorUtils.getBlockByName( block.title )
```

Comment:
I think just for correctness it would be better to use `toBeVisible` because presence on the page doesn't account for whether it can be seen?

Reply:
Make total sense! I updated my PR to follow your suggestion in [`1557b49` (#43392)](https://github.com/woocommerce/woocommerce/pull/43392/commits/1557b495f2be6f5fa655ee17dc2e219fae9940a3). Can you please take another look at this?